claat/render/md.go

```diff
@@ -16,6 +16,8 @@ package render
 
 import (
 	"bytes"
+	"fmt"
+  htmlTemplate "html/template"
```

Comment:
I think you can import it as is, without renaming: import ( "html/template" )
there's no other "template" in this file, is there?